test/test-ElementFactoryTest.cc

```diff
@@ -0,0 +1,73 @@
+#include "catch.hpp"
+
+#include "refract/Visitors.h"
+#include "refract/Element.h"
+
+#include "RefractElementFactory.h"
+
+
+using namespace refract;
+using namespace drafter;
+
+TEST_CASE("Create empty primitive element","[ElementFactory]") {
```

Comment:
There are in `test-ApplyVisitorTest.cc` in same position.
But we should finally create clang-format config and add into repo